bundle/permissions/workspace_root.go

```diff
@@ -77,3 +84,32 @@ func getWorkspaceObjectPermissionLevel(bundlePermission string) (workspace.Works
 		return "", fmt.Errorf("unsupported bundle permission level %s", bundlePermission)
 	}
 }
+
+// checkWorkspaceRootPermissions checks that if permissions are set for the workspace root, and workspace root starts with /Workspace/Shared, then permissions should be set for group: users
+func checkWorkspaceRootPermissions(b *bundle.Bundle) diag.Diagnostics {
+	var diags diag.Diagnostics
+	if len(b.Config.Permissions) == 0 {
+		return nil
+	}
+
+	if !strings.HasPrefix(b.Config.Workspace.RootPath, "/Workspace/Shared") {
+		return nil
+	}
+
+	allUsers := false
+	for _, p := range b.Config.Permissions {
+		if p.GroupName == "users" && p.Level == CAN_MANAGE {
+			allUsers = true
+		}
+	}
+
+	if !allUsers {
+		diags = diags.Append(diag.Diagnostic{
+			Severity: diag.Warning,
+			Summary:  "workspace_root_permissions",
```

Comment:
This can be something like "the bundle root path (...) is writable by all workspace users".